.eslintrc.json

```diff
@@ -17,6 +17,10 @@
 		"eslint-plugin/require-meta-docs-url": "off",
 		"eslint-plugin/test-case-property-ordering": [ "error",
 			[ "code", "options", "parserOptions", "settings", "errors", "output" ]
-		]
+		],
+		"prefer-arrow-callback": "error",
```

Comment:
the first three are default upstream.

Reply:
Happy to remove. Why did you add them, though?